docs/reference/user/provisioning/scim-via-curl.md

```diff
@@ -0,0 +1,172 @@
+# Using the SCIM API with curl {#RefScimViaCurl}
```

Comment:
I would recommend changing the path to "docs/reference/provisioning/scim-via-curl.md" because the `reference/user/provisioning` hierarchy seems needlessly deep.

(NB: I intended "user/" to be "parts of the user concept", not "everything related to users". But then I actually expect us to change the hierarchy several times, anyway.)